### PR TITLE
[RW-7340][risk=no] Run e2e nightly tests in staging

### PR DESCRIPTION
**Pull request by aweng98**
Enable e2e nightly tests to run in staging.

### CODE REVIEW

File: e2e/app/component/runtime-panel.ts [hunk 1]
```diff
@@ -228,8 +228,6 @@ export default class RuntimePanel extends BaseHelpSidebar {
     logger.info('Deleting runtime');
     await this.open();
     await this.clickButton(LinkText.DeleteEnvironment);
-    // Select "Delete gce runtime and pd" radiobutton.
-    await RadioButton.findByName(this.page, { dataTestId: 'delete-runtime' }).select();
```
Comment (aweng98) [inline e2e/app/component/runtime-panel.ts:232]: Button is gone in staging.

File: e2e/app/page/notebook-page.ts [hunk 2]
```diff
@@ -510,8 +509,6 @@ export default class NotebookPage extends NotebookFrame {
 
     // Click 'delete environment' then Delete buttons.
     await runtimePanel.clickButton(LinkText.DeleteEnvironment);
-    // Select "Delete gce runtime and pd" radiobutton.
-    await RadioButton.findByName(this.page, { dataTestId: 'delete-runtime' }).select();
```
Comment (aweng98) [inline e2e/app/page/notebook-page.ts:514]: Button is gone in staging.

File: e2e/tests/nightly/gce-to-dataproc.spec.ts [hunk 3]
```diff
@@ -72,7 +76,6 @@ describe('Updating runtime compute type', () => {
 
     // Delete runtime
     await notebook.deleteRuntime();
-    await notebook.deleteUnattachedPd();
```
Comment (aweng98) [inline e2e/tests/nightly/gce-to-dataproc.spec.ts:75]: `Delete Persistent Disk` button is not found in workspace with CT in staging.

File: e2e/resources/workbench-config.ts [hunk 4]
```diff
@@ -31,6 +31,7 @@ const local: IEnvConfig = {
   ADMIN_TEST_USER: process.env.ADMIN_TEST_USER || 'puppeteer-admin-1@fake-research-aou.org',
   DEFAULT_CDR_VERSION_NAME: 'Synthetic Dataset v3',
   ALTERNATIVE_CDR_VERSION_NAME: 'Synthetic Dataset v3 with WGS',
+  CONTROLLED_TIER_CDR_VERSION_NAME: 'Synthetic Dataset in the Controlled Tier',
```
Comment (calbach) [inline e2e/resources/workbench-config.ts:34]: this doesn't have WGS right now, so I would expect nightly to fail in test without https://precisionmedicineinitiative.atlassian.net/browse/RW-7207

File: e2e/tests/nightly/gce-to-dataproc.spec.ts (hunk 3 above)
Comment (calbach) [reply e2e/tests/nightly/gce-to-dataproc.spec.ts:75]: Per standup, likely should make this a boolean in workbench-config, with a note to keep synchronized with the enablePersistentDisk feature flag

The alternative would be to read this directly from the API config, but that would be setting a new precedent

File: e2e/app/page/workspaces-page.ts [hunk 5]
```diff
@@ -80,18 +81,31 @@ export default class WorkspacesPage extends AuthenticatedPage {
    */
   async createWorkspace(
     workspaceName: string,
-    cdrVersionName: string = config.DEFAULT_CDR_VERSION_NAME,
-    billingAccount: string = UseFreeCredits,
-    reviewRequest = false
+    opts: {
```
Comment (calbach) [inline e2e/app/page/workspaces-page.ts:84]: cool

File: e2e/resources/workbench-config.ts (hunk 4 above)
Comment (aweng98) [reply e2e/resources/workbench-config.ts:34]: Yes, nightly tests fail. right now, the PR is blocked by RW-7207.

File: e2e/resources/workbench-config.ts [hunk 6]
```diff
@@ -31,7 +31,9 @@ const local: IEnvConfig = {
   ADMIN_TEST_USER: process.env.ADMIN_TEST_USER || 'puppeteer-admin-1@fake-research-aou.org',
   DEFAULT_CDR_VERSION_NAME: 'Synthetic Dataset v3',
   ALTERNATIVE_CDR_VERSION_NAME: 'Synthetic Dataset v3 with WGS',
```
Comment (calbach) [inline e2e/resources/workbench-config.ts:33]: What are the requirements around ALTERNATIVE_CDR_VERSION_NAME, just that it's an older version? If so, can we rename it to something like OLD_CDR_VERSION_NAME.

I will need to send another config update - my most recent one archived all but two versions, and I think we now need 3 to support puppeteer

**Comment by calbach** [comment]
OK, things seem stable now.

**Pull request merged**